metrics/deploy/prometheus-ocs-rules.yaml

```diff
@@ -449,3 +449,15 @@ spec:
       for: 5m
       labels:
         severity: warning
+    - alert: DuplicateStorageAutoScalerCRsFound
+      annotations:
+        description: |-
+          Duplicate StorageAutoScaler CRs found in the namespace {{ $labels.namespace }} trying to scale device class {{ $labels.device_class }} under storage cluster {{ $labels.storagecluster }}.
+        message: Duplicate StorageAutoScaler CRs found.
+        runbook_url: https://github.com/openshift/runbooks/blob/master/alerts/openshift-container-storage-operator/DuplicateStorageAutoScalerCRsFound.md
+        severity_level: warning
+      expr: |
+        count by (namespace, managedBy, storagecluster, device_class) (ocs_storageautoscaler_capacity_reached) > 1
+      for: 1m
+      labels:
+        severity: warning
```

Comment:
@aruniiird  @umangachapagain  wont this be critical? As we have two crs that are scaling at the same time

Reply:
No since you'd be marking duplicates as invalid and not scaling those.

Reply:
I see, got your point